reference/conanfile/methods.rst

```diff
@@ -894,6 +894,37 @@ even if it matches with the default of the compiler being used:
 
 Same behavior applies if you use the deprecated setting ``cppstd``.
 
+
+CompatiblePackage
+^^^^^^^^^^^^^^^^^
+The ``package_id()`` method serves to define the "canonical" binary package ID, the identifier of the binary that correspond to the
+input configuration of settins and options. This canonical binary package ID will be always computed, and Conan will check for its
+existence to be downloaded and installed.
+
+If the binary of that package ID is not found, Conan allows to define an ordered list of compatible package IDs, of other configurations
+that are fully binary compatible and can be used as a fallback. The syntax to do this is:
+
+.. code-block:: python
+
+    from conans import ConanFile, CompatiblePackage
+
+    class Pkg(ConanFile):
+        settings = "os", "compiler", "arch", "build_type"
+
+        def package_id(self):
+            if self.settings.compiler == "gcc" and self.settings.compiler.version == "4.9":
+                compatible_pkg = CompatiblePackage(self)
+                compatible_pkg.settings.compiler.version = "4.8"
+                self.compatible_packages.append(compatible_pkg)
+
+This will define that if we try to install this package with ``gcc 4.9`` and there is not binary for that configuration, it will check
```

Comment:
```suggestion
This will define that, if we try to install this package with ``gcc 4.9`` and there isn't a binary available for that configuration, Conan will check
```